lighthouse-core/lib/page-functions.js

```diff
@@ -218,6 +218,40 @@ function getNodeSelector(node) {
   return parts.join(' > ');
 }
 
+/**
```

Comment:
can you add a comment for this function? Basically trying to provide a human recognizable label for the given element. Falls back to the tagName if no useful label is found.